src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,130 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+type Props = {
+  todos: Todo[],
+  selectedUserId: number,
+  selectUser: (id: number) => void
+};
+
+type State = {
+  visibleTodos: Todo[],
+  searchQuery: string;
+};
+
+export class TodoList extends React.Component<Props, State> {
+  state = {
+    visibleTodos: this.props.todos,
+    searchQuery: '',
+  };
+
+  componentDidMount() {
+    this.getVisible();
+  }
```

Comment:
When you render component first time, state has todos from props, because u write it at 18th line, but when it were rendered, you use componentDidMount and rewrite equal todos, you don't need it